type.py

```diff
@@ -2,9 +2,12 @@
 from enum import Enum
 from sklearn.base import ClassifierMixin
 from typing import Callable, List, Tuple
+import pandas as pd
+from transformers.training_args import TrainingArguments
 
-from transformers import TrainingArguments
 
+TrainDataset = pd.DataFrame
```

Comment:
Why is this needed? So we can switch out the type of the dataset afterwards?

Reply:
right now it's completely implicit that the furst one is train and the second one is the test dataset - but the distinction is important